examples/AnalyticsTelemetrics/README.md

```diff
@@ -0,0 +1,101 @@
+# AnalyticsTelemetry #
+
+This example MAUI application shows how to use Realm for Analytics or Telemetry application, by using [Data Ingest](https://www.mongodb.com/docs/realm/sdk/dotnet/sync/asymmetric-sync/), and how to visualize the captured data using [Charts](https://www.mongodb.com/docs/charts/). Data Ingest can be used when you need to sync data unidirectionally to Atlas, for example with telemetry in IoT applications, or you need to write immutable data and do not need conflict resolution, for example when logging analytics events. 
+
+In order to sync data unidirectionally to Atlas with the .NET Realm SDK, the objects that need to be synced must be declared as asymmetric object, using the [IAsymmetricObject](https://www.mongodb.com/docs/realm-sdks/dotnet/latest/reference/Realms.IAsymmetricObject.html) interface. Asymmetric objects follow some specific rules:
+- Asymmetric objects can have embedded object ([IEmbeddedObject](https://www.mongodb.com/docs/realm-sdks/dotnet/latest/reference/Realms.IEmbeddedObject.html)) properties, but no other asymmetric object or realm object ([IRealmObject](https://www.mongodb.com/docs/realm-sdks/dotnet/latest/reference/Realms.IRealmObject.html)) properties.
+- It is not possible to query, modify or delete asymmetric objects. Once an asymmetric object has been added to a realm, it cannot be accessed anymore, and the C# object itself cannot be used as well. 
+- Asymmetric objects can only be used with [Flexible Sync](https://www.mongodb.com/docs/realm/sdk/dotnet/sync/flexible-sync/). As they cannot be queried, you do not need to add subscriptions to asymmetric objects.   
+
+Aside from the aforementioned rules, asymmetric objects can be created and added to a realm in the same way as any other realm object. 
+
+## Configuration ##
```

Comment:
I think it would make more sense to link to the Microsoft documentation for the installation, so we're sure that they're always updated.

Reply:
I think we can still provide the minimum setup commands to avoid having to read an entire page of documentation to do something simple. We can still link to the MS docs page for users that need more details, but - for example - if I'm a WPF developer and don't know too much about MAUI, it should still be easy for me to get the app up and running and see something happening.

Reply:
Done